pkg/controller/batchrelease/utils/utils.go

```diff
@@ -14,18 +14,27 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package batchrelease
```

Comment:
original file name batchrelease_util.go is more clear, especially in the logging entries. plz consider keep the original name

Reply:
This is to solve the problem of circular reference.